delfin/task_manager/scheduler/schedule_manager.py

```diff
@@ -75,6 +75,20 @@ def on_node_join(self, event):
                 self.task_rpcapi.remove_job(self.ctx, task['id'],
                                             task['executor'])
             distributor.distribute_new_job(task['id'])
+        failed_tasks = db.failed_task_get_all(self.ctx)
+        for failed_task in failed_tasks:
+            # Get the parent task executor
+            task = db.task_get(self.ctx, failed_task['task_id'])
```

Comment:
Use the failed_task id to get the normal task, so failed task and normal task have the same id?

Reply:
task_id field in failed_task represent parent task_id, we use this to get the parent task for the failed_task